*** tests/A-12-03-M-ok.out	2021-11-07 16:04:59.226995000 +0000
--- tests/A-12-03-M-ok.outhyp	2021-11-13 00:52:52.498404110 +0000
***************
*** 95,108 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador do parceiro: Identificador do produto: Preço: Quantidade: Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
  3 - Registar venda
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador do parceiro: Identificador do produto: Preço: Quantidade: Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
  3 - Registar venda
--- 95,119 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador do parceiro: Identificador do produto: Preço: Quantidade: Registar receita? Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
  3 - Registar venda
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Número inválido!
! Escolha uma opção: Número inválido!
! Escolha uma opção: Opção inválida!
! Gestão de Transacções
! 1 - Visualizar
! 2 - Registar desagregação
! 3 - Registar venda
! 4 - Registar compra
! 5 - Receber pagamento de venda
! 0 - Sair
! Escolha uma opção: Opção inválida!
! Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
  3 - Registar venda
***************
*** 120,127 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: -60000
! Saldo contabilístico: -60000
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 131,137 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: -30000.0
  Menu Principal
  1 - Abrir
  2 - Guardar
