*** tests/A-18-02-M-ok.out	2021-11-10 23:23:51.359926000 +0000
--- tests/A-18-02-M-ok.outhyp	2021-11-17 00:50:20.090773550 +0000
***************
*** 71,102 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: VENDA|0|S1|SAL|10|1000|1000|10
! Gestão de Parceiros
! 1 - Mostrar parceiro
! 2 - Mostrar parceiros
! 3 - Registar parceiro
! 4 - Activa/desactivar notificações de um produto
! 5 - Mostrar compras com parceiro
! 6 - Mostrar vendas (e desagregações) com parceiro
! 0 - Sair
! Escolha uma opção: Identificador do parceiro: Gestão de Parceiros
! 1 - Mostrar parceiro
! 2 - Mostrar parceiros
! 3 - Registar parceiro
! 4 - Activa/desactivar notificações de um produto
! 5 - Mostrar compras com parceiro
! 6 - Mostrar vendas (e desagregações) com parceiro
! 0 - Sair
! Escolha uma opção: Menu Principal
! 1 - Abrir
! 2 - Guardar
! 3 - Mostrar Data
! 4 - Avançar Data
! 5 - Gestão de Produtos
! 6 - Gestão de Parceiros
! 7 - Gestão de Transacções
! 8 - Menu de Consultas
! 9 - Ver Saldo Global
! 0 - Sair
! Escolha uma opção: 
\ No newline at end of file
--- 71,74 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: 
\ No newline at end of file
