*** tests/A-20-06-M-ok.out	2021-11-18 10:37:04.077363100 +0000
--- tests/A-20-06-M-ok.outhyp	2021-11-18 17:28:30.088403700 +0000
***************
*** 65,71 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: M1|Rohit Figueiredo|New Delhi, India|NORMAL|1818|0|202|182
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 65,71 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: M1|Rohit Figueiredo|New Delhi, India|NORMAL|1818|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
