*** tests/A-20-22-M-ok.out	2021-11-18 17:42:16.390816473 +0000
--- tests/A-20-22-M-ok.outhyp	2021-11-18 23:04:03.816104336 +0000
***************
*** 140,146 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|ELITE|26000|0|3600|3550
  M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
--- 140,146 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|ELITE|35500|0|3600|3550
  M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
