*** tests/A-20-15-M-ok.out	2021-11-18 10:37:05.442877700 +0000
--- tests/A-20-15-M-ok.outhyp	2021-11-18 17:28:35.412560600 +0000
***************
*** 68,75 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|SELECTION|23000|0|2410|2300
! M2|Toshiba|Tokyo, Japan|SELECTION|2100|0|210|210
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
--- 68,75 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|SELECTION|23000|0|0|0
! M2|Toshiba|Tokyo, Japan|SELECTION|2100|0|0|0
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
***************
*** 146,153 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|ELITE|26000|0|2710|2600
! M2|Toshiba|Tokyo, Japan|SELECTION|2100|0|210|210
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
--- 146,153 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|ELITE|26000|0|0|0
! M2|Toshiba|Tokyo, Japan|SELECTION|2100|0|0|0
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
